turbinia/config/turbinia_config.py

```diff
@@ -82,24 +81,23 @@
 # GCP AND GCE #
 ###############
 
-PROJECT = None
-ZONE = None
-INSTANCE = None
-DEVICE_NAME = None
-SCRATCH_PATH = None
+PROJECT = 'None'
+ZONE = 'None'
+INSTANCE = 'None'
+DEVICE_NAME = 'None'
+SCRATCH_PATH = 'None'
 # GCS bucket that has Turbinia-specific scripts and can be used to store logs.
-BUCKET_NAME = None
-PSQ_TOPIC = 'turbinia-psq'
```

Comment:
Why not keep these?  These aren't globally unique like GCS buckets, etc.

Reply:
Cool, can do.